tests/integration/test_data/cached_dependencies.yaml

```diff
@@ -185,6 +197,18 @@ gomod_cached_deps:
       replaces: null
       type: "gomod"
       version: "DEP_VERSION"
+    - name: "golang.org/x/text"
+      replaces: null
+      type: "gomod"
+      version: "v0.0.0-20170915032832-14c0d48ead0c"
+    - name: "rsc.io/quote"
+      replaces: null
+      type: "gomod"
+      version: "v1.5.2"
+    - name: "rsc.io/sampler"
+      replaces: null
+      type: "gomod"
+      version: "v1.3.0"
```

Comment:
I don't really understand why these show up on both `go-package` and `gomod` parts...

Reply:
See if this explains it: https://github.com/release-engineering/cachito/#go-package-level-dependencies-and-the-go-package-cachito-package-type

The most important part is probably this
> A directory that contains both `go.mod` and `.go` files is both a module and a package.

Reply:
But I guess the question is why they didn't show up before, huh?

Reply:
I'm thinking it's because the commit would remove the `rsc.io` dep, so these would not show in the resulting packages.

Reply:
Yeah, probably. It's still a bit strange though, if go 1.15 fixed the missing `require` automatically, I would expect the modules to be properly listed. If that was not the case, we could have been reporting wrong data about the dependencies that Cachito provided. At least we would have listed the packages, only the modules would be missing.

It's good to know that go 1.16 fixes that. It may also break builds that used to work, but I wouldn't worry about that since we make the audacious assumption that users will get their package in order before trying to process it with Cachito.